radixdlt-kotlin/src/main/kotlin/com/radixdlt/client/core/Bootstrap.kt

```diff
@@ -12,6 +12,10 @@ enum class Bootstrap(private val config: RadixUniverseConfig, private val discov
         RadixUniverseConfigs.alphanet,
         PeersFromNodeFinder("https://alphanet.radixdlt.com/node-finder", 443)
     ),
+    ALPHANET2(
+        RadixUniverseConfigs.alphanet,
```

Comment:
This is wrong. It should read `RadixUniverseConfigs.alphanet2` (and that configuration should, of course be added as well).